cuda_core/cuda/core/experimental/_event.py

```diff
@@ -47,8 +47,25 @@ class Event:
     the last recorded stream.
 
     Events can be used to monitor device's progress, query completion
-    of work up to event's record, and help establish dependencies
-    between GPU work submissions.
+    of work up to event's record, help establish dependencies
+    between GPU work submissions, and record the elapsed time on GPU:
```

Comment:
The text needs to name the expected units for the elapsed time. milliseconds, I believe.

Reply:
Yes indeed, like this?
```suggestion
    between GPU work submissions, and record the elapsed time (in milliseconds)
    on GPU:
```